### PR TITLE
Works Section of My Portfolio

### PR DESCRIPTION
**Pull request by maov19**
- Add projects portfolio
- Add about section

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -107,3 +107,266 @@ a:link {
...
+.works {
+  margin-top: 150px;
+}
+
+.project {
+  display: grid;
+  grid-template-columns: auto auto auto;
+  grid-template-rows: auto;
+  margin-left: 30px;
+}
```
Comment (Shedrack-Sunday) [inline style.css]: - [x] You did a great job on the project, however, the instruction is to use a grid for the main `section` that wraps your `work` section, then use flexbox to display the cards. based on your design, the section that wrapped them has the class of `works`, which should be a grid, then the different cards in class `project` should be displayed with flexbox.
- [x] You did put in a lot o effort 👏 , but let's stick to the project requirement 
# Instruction:
image

File: style.css [hunk 2]
```diff
@@ -107,3 +107,266 @@ a:link {
...
+  font-style: normal;
+  font-weight: 5000;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
```
Comment (Shedrack-Sunday) [inline style.css]: - [x] Kindly refer to the [FIGMA](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=vWKp7eO7bXR4dhXY-0) to tally the font. yours is slightly different. find the comparison below :

# Figma Design:
image

# Your design:
image

File: style.css [hunk 3]
```diff
@@ -107,3 +107,266 @@ a:link {
...
+  height: 146px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
```
Comment (Shedrack-Sunday) [inline style.css]: - [x] Nice one with the styling of the paragraph about the "About " Section. however, you need to adjust the font to the figma requirements 
# Figma:
image

# Your Design:
image

File: style.css [hunk 4]
```diff
@@ -107,3 +107,266 @@ a:link {
...
+}
+
+.skills {
+  display: grid;
+  grid-template-columns: auto;
+  gap: 8px;
+}
```
Comment (Shedrack-Sunday) [inline style.css]: - [x] There are quite a few changes to make in this section to get you going and tally with the figma design.
- A future functionality will allow you close or open the listing of the technologies, and that's why those arrow bars are present. Your design is void of it, and I advise you to fix it. 😃 
- The margin between the `languages` and technologies in your design is a little bit different. please fit it 👍 
- The margin on the left of your design needs to be adjusted a little, to look just as the design is given. I've compared the two designs for you. do make the changes to compliment the brilliant work you've done so far of which I must commend you for 👏 

# Figma
image

# Your designs 
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @maov19  , 

Great work on the project so far👏 
You've done well implementing most of the requested features, but there are still some which aren't been addressed yet.


 ## Highlights
 -  [ ] Nice workflow 👍 
 -   [ ] Proper Pull request ✔️ 
 -  [ ] Good commit history ✅ 


 ## Required Changes ♻️
 
- [ ] I pointed out some design changes you need to make, which are slight variations from the figma design 


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (maov19) [reply style.css]: Fonts were modified to comply with figma styling requirements (re-downloaded font with adequate weights)

File: style.css (hunk 3 above)
Comment (maov19) [reply style.css]: Fonts stylings were re-downloaded from google fonts to include figma styling requirements.

File: style.css (hunk 4 above)
Comment (maov19) [reply style.css]: - Add arrow bars, as requested.
- Add margin-bottom between languages and technologies
- Left margin was fixed, as required (aligned with the text of the button above)

File: style.css (hunk 2 above)
Comment (Shedrack-Sunday) [reply style.css]: Alright @maov19, kindly send for a review on the dashboard. Good job 👏

File: style.css (hunk 1 above)
Comment (maov19) [reply style.css]: The styling was corrected. Grid was styling was setup to works section and flexbox to its projects.

**Comment by maov19** [comment]
Dear reviewer,
The issues detailed in the first review were solved. Please let me know if there's anything else I should work on.
I'll appreciate your fast response, the deadline is today. 
Thanks!

**Review by roshan-bajgain — APPROVED** [review]
Hi @maov19, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**